samples/sid_end_device/sample.yaml

```diff
@@ -1,113 +1,63 @@
 sample:
   name: Sidewalk end device sample
   description: Sample implementing Amazon Sidewalk End Device
+common:
+  build_only: true
+  platform_allow:
+    - nrf52840dk_nrf52840
+    - nrf5340dk_nrf5340_cpuapp
+    - nrf54l15pdk_nrf54l15_cpuapp
+  integration_platforms:
+    - nrf52840dk_nrf52840
+    - nrf5340dk_nrf5340_cpuapp
+    - nrf54l15pdk_nrf54l15_cpuapp
 tests:
   sample.sidewalk.hello:
-    build_only: true
-    platform_allow:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf54l15pdk_nrf54l15_cpuapp
     extra_configs:
       - CONFIG_SID_END_DEVICE_PERSISTENT_LINK_MASK=y
-    integration_platforms:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf54l15pdk_nrf54l15_cpuapp
     tags: Sidewalk hello
 
   sample.sidewalk.hello.release:
-    build_only: true
-    platform_allow:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf54l15pdk_nrf54l15_cpuapp
     extra_args:
       CONF_FILE=prj_release.conf
     extra_configs:
       - CONFIG_SID_END_DEVICE_PERSISTENT_LINK_MASK=y
-    integration_platforms:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf54l15pdk_nrf54l15_cpuapp
     tags: Sidewalk hello
 
   sample.sidewalk.hello.ble_only:
-    build_only: true
-    platform_allow:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf54l15pdk_nrf54l15_cpuapp
     extra_configs:
       - CONFIG_SIDEWALK_SUBGHZ_SUPPORT=n
-    integration_platforms:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf54l15pdk_nrf54l15_cpuapp
     tags: Sidewalk hello BLE
 
   sample.sidewalk.hello.ble_only.release:
-    build_only: true
-    platform_allow:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf54l15pdk_nrf54l15_cpuapp
     extra_args:
       CONF_FILE=prj_release.conf
     extra_configs:
       - CONFIG_SIDEWALK_SUBGHZ_SUPPORT=n
-    integration_platforms:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf54l15pdk_nrf54l15_cpuapp
     tags: Sidewalk hello BLE
 
   sample.sidewalk.demo:
-    build_only: true
-    platform_allow:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf54l15pdk_nrf54l15_cpuapp
     extra_args:
       OVERLAY_CONFIG="overlay-demo.conf"
```

Comment:
change to `EXTRA_CONF_FILE`

Reply:
Is `OVERLAY_CONFIG` deprecated? I see it in many samples and documentation articles